src/api/routes/tasks.ts

```diff
@@ -6,15 +6,55 @@ const router = express.Router();
 // GET request /tasks route
 router.get("/", (req: Request, res: Response, next: NextFunction) => {
   try {
-    console.log(data);
-    res.status(200).json(data);
+    let count: any = req.query.count;
+    // console.log("/-count: ", count);
+    if (count) {
+      let result = [];
+      for (let i: any = 0; i < parseInt(count); i++) {
+        result.push(data.data[i]);
+      }
```

Comment:
What if `count` is greater than `data.data.length`? You need to make sure `i` doesn't exceed the maximum size of the array.

Reply:
Do the same as I mentioned below. That would be sufficient. You don't need to throw an error.